.github/CONTRIBUTING.md

```diff
@@ -129,6 +135,16 @@ You can run the unit tests for KerasCV by running:
 pytest keras_cv/
 ```
 
+### Tests that require custom ops
+For tests that require custom ops, you'll have to compile the custom ops and make them available to your local Python code:
+```shell
+python build_deps/configure.py
+bazel build keras_cv/custom_ops:all
+cp bazel-bin/keras_cv/custom_ops/*.so keras_cv/custom_ops/
+```
+
+Tests which use custom ops are disabled by default, but can be run by setting the environment variable `TEST_CUSTOM_OPS=true`.
```

Comment:
can this be True by default? Or what's the disadvantage to support testing things built on top of custom ops?

Reply:
Testing these ops requires building the library from source, which a contributor would have to do iff they are updating custom ops.

It's pretty trivial to opt-in the CI tests to running these tests, so Luke figured it would be nicer to turn these off by default to reduce friction for the vast majority of contributors who won't care about the custom ops.

I'm fine either way.

Reply:
I'd like to make this disabled by default  to prioritize OSS contributors?  

lets discuss in weekly?